### ISSUE TITLE
Add parameter to do authorization

### ISSUE THREAD

**Issue by Sraw** [oilervoss/transmission#3]
Sometimes transmission-daemon will have a rpc authorization. In this case, transmission-remote need `-n un:pw` to connect to transmission.

We can modify the script by ourselves but it would be better to support this natively.

**Comment by darkrain88** [comment]
yes close authorization. run remote -l will find the hint of error
